### PR TITLE
Add demo for VCX Nodejs Wrapper

### PR DESCRIPTION
**Pull request by Patrik-Stas**
Signed-off-by: Patrik Stas <patrik.stas@gmail.com>

### CODE REVIEW

**Comment by jovfer** [comment]
@Artemkaaas please review

File: vcx/wrappers/node/README.md [hunk 1]
```diff
@@ -33,3 +33,21 @@ npm run doc-gen
...
+```
+- Start [Dummy Cloud Agent](../../dummy-cloud-agent)
+- Run Faber agent, representing an institution
+```
+npm run demo:faber
+```
+- Give it a few seconds, then run Alice's agent which will connect with Faber's agent
+```
+npm run demo:alice
+```
```
Comment (Artemkaaas) [inline vcx/wrappers/node/README.md]: pool genesis transactions file is missed

Comment (Artemkaaas) [reply vcx/wrappers/node/README.md]: reflect that payment plugin is necessary

Comment (Patrik-Stas) [reply vcx/wrappers/node/README.md]: I've added more documentation now, including mention of libnullpay.

**Comment by Patrik-Stas** [comment]
I rebased on the latest master as the build was failing, but seems like it still does. Anything on my side I can do?

**Comment by jovfer** [comment]
@Patrik-Stas there is some unstable behavior on CI/CD. I will inform you if some fixes will be required from your side

**Pull request merged**